libs/assets/src/icons/svg/filterOutline.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none"
+    xmlns="http://www.w3.org/2000/svg">
+    <g id="cuida:filter-outline">
+        <g id="Group">
+            <path id="Vector" fill-rule="evenodd" clip-rule="evenodd" d="M3 7C3 6.73478 3.10536 6.48043 3.29289 6.29289C3.48043 6.10536 3.73478 6 4 6H20C20.2652 6 20.5196 6.10536 20.7071 6.29289C20.8946 6.48043 21 6.73478 21 7C21 7.26522 20.8946 7.51957 20.7071 7.70711C20.5196 7.89464 20.2652 8 20 8H4C3.73478 8 3.48043 7.89464 3.29289 7.70711C3.10536 7.51957 3 7.26522 3 7ZM5 11.5C5 11.2348 5.10536 10.9804 5.29289 10.7929C5.48043 10.6054 5.73478 10.5 6 10.5H18C18.2652 10.5 18.5196 10.6054 18.7071 10.7929C18.8946 10.9804 19 11.2348 19 11.5C19 11.7652 18.8946 12.0196 18.7071 12.2071C18.5196 12.3946 18.2652 12.5 18 12.5H6C5.73478 12.5 5.48043 12.3946 5.29289 12.2071C5.10536 12.0196 5 11.7652 5 11.5ZM8 16C8 15.7348 8.10536 15.4804 8.29289 15.2929C8.48043 15.1054 8.73478 15 9 15H15C15.2652 15 15.5196 15.1054 15.7071 15.2929C15.8946 15.4804 16 15.7348 16 16C16 16.2652 15.8946 16.5196 15.7071 16.7071C15.5196 16.8946 15.2652 17 15 17H9C8.73478 17 8.48043 16.8946 8.29289 16.7071C8.10536 16.5196 8 16.2652 8 16Z" fill="currentColor"/>
+        </g>
```

Comment:
@Davit-BetterAngels @vecchp @mikefeldberg 

FYI - whenever adding an `svg` file to `libs/assets` we'd typically want the ability to change the color with CSS. To do so we'd need to make sure that `fill="currentColor"`. 

Typically the SVGs from Figma will have a hard-coded color, which we typically don't want (except for branded stuff etc)

Reply:
Yah, we've gone through that exercise a few times we either export it to not specify a color or manually edit the svg to have currentColor.